examples/package.json

```diff
@@ -36,7 +36,7 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "styled-components": "^5.2.0",
-    "three": "link:../viewer/node_modules/three"
+    "three": "^0.122.0"
```

Comment:
@larsmoa why to remove the link? We just will run into conflicts between versions as we had before.

Reply:
@maksnester : oops, should we have a similar link in documentation btw?

Reply:
Sure, we link reveal and sdk already. Probably make sense to do the same for three.js too.